games-arcade/ccleste/ccleste-1.4.0.recipe

```diff
@@ -0,0 +1,60 @@
+SUMMARY="A classic platform game"
+DESCRIPTION="Celeste Classic is the original free version of Celeste \
+, created in 2016. It was written for the Pico-8 virtual console. \
+The game can be found on Pico-8's website and on itch.io. \
+The original code for the game was written in a form of the Lua \
+programming language tailored to the Pico-8, although the version of \
+Celeste Classic found inside the full Celeste game is written in C#."
+
+HOMEPAGE="https://github.com/lemon32767/ccleste"
+COPYRIGHT="lemon32767"
+LICENSE=""
```

Comment:
License can't be found anywhere.
```diff
-LICENSE=""
+LICENSE="Unknwon"
```
This means you can only build if this is localy changed to something else and wont end up in the depot.